Library/Homebrew/test/livecheck/skip_conditions_spec.rb

```diff
@@ -265,56 +265,56 @@
   end
 
   describe "::skip_conditions" do
-    context "a deprecated formula without a livecheckable" do
+    context "when the formula is deprecated without a livecheckable" do
```

Comment:
I think `"when a formula without a livecheckable is deprecated"` (and similarly for the other tests) conveys the meaning better.

Reply:
Thanks, I updated all of those.